client/src/legacy/LeftAndMain.EditForm.js

```diff
@@ -656,6 +656,30 @@ $.entwine('ss', function($){
     }
   });
 
+  // Check if there is is focusable content inside tab panels
+  // otherwise set tabindex="0" so that the panel itself can be focused
+  // See point 4 under notes on https://www.w3.org/WAI/ARIA/apg/patterns/tabs/
+  $('.cms-edit-form .tab-pane').entwine({
+    onmatch: function() {
+      // This selector is duplicated in TabItem.js - keep in sync
+      const cssSelector = [
+        'a[href]',
+        'button:not([disabled])',
+        'input:not([disabled]):not([type="hidden"])',
+        'select:not([disabled])',
+        'textarea:not([disabled])',
+        '[tabindex]:not([tabindex="-1"])',
+        'iframe',
+        'object',
+        'embed',
```

Comment:
Are these focusable? I wouldn't have thought so, but I could easily be wrong.

Reply:
According to https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Global_attributes/tabindex they are

Based on that article I'm also added 'summary' and removed 'embed'. I did not add area or frame since they do not seem applicable in this context